PA193_mnemonic_Slytherin/mnemonic.py

```diff
@@ -111,63 +182,7 @@ def _entropy2mnemonic(entropy: bytes) -> str:
     return ' '.join(words)
 
 
-def _mnemonic2entropy(mnemonic: str) -> bytes:
-    """Convert mnemonic phrase to entropy using dictionary.
-    :raises ValueError: on invalid parameters
-    :rtype: bytes
-    :return: Entropy
-    """
-    if not isinstance(mnemonic, str):
-        raise ValueError('invalid mnemonic')
-    
-    words = mnemonic.split()
-    l = len(words)
-    if not l in (12, 15, 18, 21, 24):
-        raise ValueError('invalid mnemonic')
-
-    _, word_dict = __get_dictionary()
-    try:
-        indexes = [word_dict[word] for word in words]
-    except KeyError:
-        raise ValueError('invalid mnemonic')
-
-    # Concatenate indexes into single 
-    indexes_bin = sum([indexes[-i - 1] << i * 11 for i in reversed(range(l))])
-
-    # Number of bits entropy is shifted by
-    shift = l * 11 // 32
-
-    checksum = indexes_bin & (pow(2, shift) - 1)
-    entropy_bin =  indexes_bin >> shift
-    entropy = entropy_bin.to_bytes((l * 11 - shift) // 8, byteorder='big')
-
-    # Check correctness
-    check = __get_entropy_checksum(entropy, shift)
-    if check != checksum:
-        raise ValueError('invalid mnemonic')
-    
-    return entropy
-
-
-def is_valid_entropy(entropy: bytes) -> bool:
-    """Check whether provided bytes represent a valid entropy according to BIP39.
-    https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki
-
-    > The mnemonic must encode entropy in a multiple of 32 bits. With more entropy security is
-    > improved but the sentence length increases. We refer to the initial entropy length as ENT.
-    > The allowed size of ENT is 128-256 bits.
-    > https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#generating-the-mnemonic
-    """
-    return len(entropy) in list(range(16, 32+1, 4))
-
-
-def is_valid_seed(seed: bytes) -> bool:
-    """Check whether provided bytes represent a valid seed.
-    """
-    return isinstance(seed, bytes) and len(seed) == SEED_LEN
-
-
-def _secure_seed_compare(expected_seed: bytes, actual_seed: bytes) -> bool:
+def _secure_seed_compare(expected_seed: Seed, actual_seed: Seed) -> bool:
```

Comment:
Could be incorporated as `__eq__` and `__ne__` methods of `Seed`.

Reply:
makes sense, then we also dont have to talk about these https://github.com/mvondracek/PA193_mnemonic_Slytherin/pull/36#discussion_r336780978 further :D